src/openrct2/core/DataSerialiserTraits.h

```diff
@@ -31,14 +31,14 @@
 #include <sstream>
 #include <stdexcept>
 
-template<typename T> struct DataSerializerTraits_t
+template<typename T> struct DataSerializerTraitsT
```

Comment:
This rename looks a bit weird - shouldn’t it remain `_t`? Same for `_enum` below.

Reply:
Both of these were just place holders as i needed something to be able to do some SIFNAE to handle enums. I never liked the _enum so thought just make it Enum.